File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLComboBoxProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLComboBoxProps
    {
        public String CanvasID;
        public String WindowID;
        public String TextAreaWindowID;
        public String ButtonWindowID;
        public String ListAreaWindowID;
        public String VScrollBarWindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String SelectedID;
        public String TextAreaTextColor;
        public String TextAreaTextHeight;
        public String TextAreaFontString;
        public String ListAreaTextColor;
        public String ListAreaTextHeight;
        public String ListAreaFontString;
        public Object Tag;
        public String TabStopIndex; 

        CCLComboBoxProps()
        {
            Data = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLComboBoxProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLComboBoxProps
    {
        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int TextAreaWindowID { get; set; }
        public int ButtonWindowID { get; set; }
        public int ListAreaWindowID { get; set; }
        public int VScrollBarWindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<string> Data { get; set; }
        public int SelectedID { get; set; }
        public string TextAreaTextColor { get; set; }
        public int TextAreaTextHeight { get; set; }
        public string TextAreaFontString { get; set; }
        public string ListAreaTextColor { get; set; }
        public int ListAreaTextHeight { get; set; }
        public string ListAreaFontString { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLComboBoxProps()
        {
            Data = new List<string>();
        }
    }
}